csun-metalab/nomi
# NAM-430 -- UI: A to Z button should look more like a button to match other UI elements like first name & last name and add the new svg

# Description
Added new SVG icon; added styling to go with the selection styling; made SVG color to current theme.
  
# Testing
1. Go to student page and check to see that no buttons break on proper view-ports.
2. Check to see that the new SVG for descending is changed to the new icon.
3. Ensure the color follows the proper theme color.
